bigquery/google/cloud/bigquery/job.py

```diff
@@ -1218,11 +1218,53 @@ def from_api_repr(cls, resource, client):
         return job
 
 
-class _AsyncQueryConfiguration(object):
-    """User-settable configuration options for asynchronous query jobs.
+class QueryJobConfig(object):
+    """Configuration options for query jobs.
 
-    Values which are ``None`` -> server defaults.
+    All properties in this class are optional. Values which are ``None`` ->
+    server defaults.
     """
+
+    def __init__(self):
+        self._properties = {}
+        # TODO(swast): Default to standard SQL syntax.
```

Comment:
You do this in Client.query_rows.  I don't think there's anything to do here.

Reply:
Removed.